include/dump_offload.hpp

```diff
@@ -152,11 +154,29 @@ class Handler : public std::enable_shared_from_this<Handler>
                 this->connection->sendStreamErrorStatus(
                     boost::beast::http::status::internal_server_error);
                 this->connection->close();
+                this->cleanupSocketFiles();
                 return;
             }
         });
     }
 
+    void cleanupSocketFiles()
+    {
+        std::error_code ec;
+        bool fileExists = std::filesystem::exists(unixSocketPath, ec);
+        if (ec)
+        {
+            this->connection->sendStreamErrorStatus(
```

Comment:
Add a error trace saying that the cleanup of this socket failed. Also print the unixSocketPath here

Reply:
done